modules/data/generic_page.components.json

```diff
@@ -3,5 +3,43 @@
         "selectorData": "body",
         "strategy": "tag",
         "groups": []
+    },
```

Comment:
Why are these in GenericPage? Are these elements in the Content Context? Kind of looks like no, but I could be wrong...

Reply:
Yeah, I think these components and methods should be in a new manifest, `amo_language_packs.components.json` and you should probably make a new class in the `page_object_addons_mozilla_org.py` POM. I really think we'll be seeing this page again...

Reply:
Sure, I will work on refactoring into a POM